src/runner/PlutoRunner/src/PlutoRunner.jl

```diff
@@ -785,6 +785,13 @@ function delete_method_doc(m::Method)
     end
 end
 
+
+if VERSION < v"1.7.0-0"
```

Comment:
```suggestion
@static if VERSION < v"1.7.0-0"
```

I don't know what this does but I've seen in in places

Reply:
Thanks! I think it doesn't matter when the `if` is at toplevel inside of a module, because it will only run once either way. It's useful in cases like this:

https://github.com/fonsp/Pluto.jl/blob/6dfbefe20800c3963f016e4f9af7663e41d57029/src/packages/PkgCompat.jl#L77

so that the `if` will run only once when you import Pluto, not every time that you call the function